### PR TITLE
update versions

### PR DESCRIPTION
**Pull request by btwonion**
mappingsVersion -> 1.19.2+build.3
loaderVersion -> 0.14.9
serializationVersion -> 1.4.0
atomicfuVersion -> 0.18.3

increase gradle version
increase mc versions

### CODE REVIEW

**Review by jakobkmar — CHANGES_REQUESTED** [review]
You forgot to run the update readme task (`processMDTemplates`).

**Comment by btwonion** [comment]
Completed

File: gradle.properties [hunk 1]
```diff
@@ -4,13 +4,13 @@ org.gradle.jvmargs=-Xmx2G
 modId=fabric-language-kotlin
 modVersion=1.8.2
```
Comment (jakobkmar) [inline gradle.properties:5]:
```suggestion
modVersion=1.8.3
```

**Review by jakobkmar — COMMENTED** [review]
and update the readme again after this

**Comment by btwonion** [comment]
done

**Review by jakobkmar — APPROVED** [review]

File: gradle.properties [hunk 2]
```diff
@@ -2,15 +2,15 @@ kotlin.code.style=official
 org.gradle.jvmargs=-Xmx2G
 
 modId=fabric-language-kotlin
-modVersion=1.8.2
+modVersion=1.8.3
 minecraftVersion=1.19
-mappingsVersion=1.19+build.1
-loaderVersion=0.14.7
+mappingsVersion=1.19.2+build.3
```
Comment (jakobkmar) [inline gradle.properties:7]: revert this since you didn't update everything to 1.19.2

Comment (btwonion) [reply gradle.properties]: fixed

**Comment by modmuss50** [comment]
Superseeded by:  https://github.com/FabricMC/fabric-language-kotlin/pull/84 Thanks a lot 👍

**Pull request closed without merging**